go.mod

```diff
@@ -64,6 +64,7 @@ require (
 )
 
 replace (
+	github.com/cri-o/ocicni => github.com/mccv1r0/ocicni v0.1.1-0.20200427184609-3afd65173207
```

Comment:
We will eventually replace this by the `cri-o-1.16` branch we pushed earlier today?

Reply:
Once that PR merges